rosdep/python.yaml

```diff
@@ -1373,6 +1373,9 @@ python-opencv:
   arch: [opencv, python2-numpy]
   debian: [python-opencv]
   fedora: [opencv-python]
+  slackware:
+    pip:
+      packages: [opencv-python]
```

Comment:
My mistake, opencv package for slackware includes the python bindings. I submitted new pull request.